index.php

```diff
@@ -1,7 +1,13 @@
 <?php 
 
 get_header(); 
-  
+if (have_posts()) :
+   while (have_posts()) :
+      the_post();
+         the_content();
+   endwhile;
+endif;
+get_sidebar();  
 get_footer();
 
 ?>
```

Comment:
As part of the standards you should always have a newline at the bottom of your files. That's why that little red icon is there.